draft-ietf-rtcweb-jsep.xml

```diff
@@ -1743,6 +1761,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>For MediaStreamTracks that are still present, the "a=msid",
               "a=ssrc", and "a=ssrc-group" lines MUST stay the same.</t>
 
+	      <t>For RtpTransceivers that are still present, the
+	      "a=rid" lines MUST stay the same.</t>
+
+	      <t>For RtpTransceivers that are still present, the
```

Comment:
The API doesn't allow adding encodings, and I don't think it can remove them either.  If you don't want to send it anymore, you set .active = false, but you don't remove it.

Do we want to allow removal?